CHANGELOG.md

```diff
@@ -235,7 +238,7 @@ All of these changes are documented in detail in the
   you're using the default skin, you'll need to add a line like the
   following:
   `<link rel="stylesheet" type="text/css"
-    href="wymeditor/skins/default/skin.css" />`
+    href="wymeditor/skins/basic/skin.css" />`
```

Comment:
See [this comment](https://github.com/wymeditor/wymeditor/pull/710#issuecomment-96103136).